src/components/Cluster/ClusterDetail/KeypairCreateModal/KeyPairCreateModal.js

```diff
@@ -119,19 +118,11 @@ const KeyPairCreateModal = (props) => {
     });
   };
 
-  const cnPrefixOrEmail = () => {
-    if (cnPrefix === '') {
-      return props.user.email;
-    }
-
-    return cnPrefix;
-  };
-
   const cnPrefixValidation = (value) => {
     let error = null;
     if (value !== '') {
       const endRegex = /[a-zA-Z0-9]$/g;
-      const regex = /^[a-zA-Z0-9][a-zA-Z0-9@\.-]*$/g;
+      const regex = /^[a-zA-Z0-9][a-zA-Z0-9@.-]*$/g;
```

Comment:
Example: 

The commit is called: "Move AddKeyPair template into a separate component"

But it is also changing this validation regex. I think in this case it's ok, I guess we don't need to escape the `.` inside a match list like that (TIL), but it makes me think a bit and would be best in a separate commit or PR even.